lib/plugins/authldap/conf/metadata.php

```diff
@@ -11,6 +11,7 @@
 $meta['deref']       = array('multichoice','_choices' => array(0,1,2,3),'_caution' => 'danger');
 $meta['binddn']      = array('string','_caution' => 'danger');
 $meta['bindpw']      = array('password','_caution' => 'danger','_code'=>'base64');
+$meta['attributes']  = array('array');
```

Comment:
Alright, my bad, but I believe DokuWiki doesn't support `array` on the GUI yet. I don't have much idea how the LDAP attributes array works now, but if possible this should be available for editing on DokuWiki's configuration manager.

Reply:
I think it does support it. Before pushing the changes I tested them in the DokuWiki instance we run at work and it worked. We are running DokuWiki Release 2018-04-22b "Greebo" in Debian 10 Buster.

Adding this in the GUI:

![ldap_array](https://user-images.githubusercontent.com/14370642/61047495-ecf4a000-a3df-11e9-9f79-16363aa820d2.png)

Creates the following line in `conf/local.php`:
```
$conf['plugin']['authldap']['attributes'] = array('cn', 'displayname', 'mail', 'givenname', 'objectclass', 'sn', 'uid', 'memberof');
```

Reply:
Thank you for testing! I only read https://www.dokuwiki.org/devel:configuration#configuration_metadata, which is outdated unfortunately.